src/librustc/session/config.rs

```diff
@@ -1074,6 +1074,9 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
         [TRACKED], "panic strategy to compile crate with"),
     incremental: Option<String> = (None, parse_opt_string, [UNTRACKED],
           "enable incremental compilation"),
+    polly: bool = (option_env!("RUSTC_FORCE_USE_POLLY").map(|_| true ).unwrap_or(false),
```

Comment:
Could you make this flag `-Z polly` instead of `-C polly`? We'll likely not insta-stabilize this flag. 

BTW, `x.map(|_| true).unwrap_or(false)` is equivalent to `x.is_some()`.

Reply:
It's a codegen option though, to me it doesn't make sense to put it in with debugging options. I've moved it anyway, but yeah.

Second point is a good one, not sure what I was thinking on that.